test/metadata_card_request_test.go

```diff
@@ -0,0 +1,61 @@
+package test
+
+import (
+	"github.com/checkout/checkout-sdk-go/metadata"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCardMetadataForCardRequest(t *testing.T) {
```

Comment:
Can we refactor these tests to use the test_cases format? Like in `customers_test.go` for example

Reply:
👍